daemon/network.go

```diff
@@ -360,6 +387,34 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 	}
 	daemon.LogNetworkEvent(n, "create")
 
+	if agent && !n.Info().Ingress() && n.Type() == "overlay" {
+
+		sandboxName := create.Name + "-sbox"
+		sb, err := c.NewSandbox(sandboxName)
+		if err != nil {
+			logrus.Errorf("Failed creating %s sandbox: %v", sandboxName, err)
+			return nil, err
```

Comment:
I don't think the error needs to be logged.  The conventional way of handling errors is to return them with `errors.Wrapf(err, "failed creating %s sandbox", sandboxName)`.

The errors should end up in the logs.